vscode/src/rbs.ts

```diff
@@ -0,0 +1,60 @@
+import * as vscode from "vscode";
+
+// This class is used to dim RBS signatures in Ruby files
+// For example, these signatures will be dimmed:
+//
+//   #: (String) -> (String | nil)
+//   #: (String) { (String) -> boolish } -> void
+//   #: return: String
+//
+// However, this will not be dimmed:
+//
+//   attr_reader :name #: String
+//
+export class RBS {
+  private decorationType: vscode.TextEditorDecorationType;
+  private disposables: vscode.Disposable[] = [];
+
+  constructor() {
+    // Create decoration type with 50% opacity
+    this.decorationType = vscode.window.createTextEditorDecorationType({
+      opacity: "0.5",
```

Comment:
@Morriar you comment said 50% but the value was `0.4`. I updated it to match the comment. Let me know if you want the other way around.